pillar.example

```diff
@@ -1,3 +1,9 @@
+## Required variables for functional pillar.example ##
+{% set host_name = salt['grains.get']('hostname') %}
+{% set id        = salt['grains.get']('get_server_id') %}
```

Comment:
(I missed your reply on this in the earlier PR.) I think those vars are undefined on purpose as a kind of hint that they should be filled out with something real later (I may have been the source of that "convention" in another formula). That said defining them with defaults so the file can be copy-and-pasted is better. 👍 

So `get_server_id` is the name of the function which sets the `server_id` Grain. I'm not familiar with that Grain or what it's used for. Looks like a kind of hashing on the Minion ID maybe? In any case, probably not what want here.

I'm not super familiar with Tomcat but glancing at the docs, if `inetdomain1` is "example.com", should `alias` then be "www.example.com" (or vice versa)?

Reply:
I personally prefer each formula to have a *good* pillar.example for more pleasant onboarding experience.  This formula installed ok on multiple OS without pillars. Once I introduced pillar.example it was disconcerting to see installation fail on every OS because of tomcat "business logic" variables are undefined - the risk is new adopters walk away from saltstack due to  sloppy user experience - I just think non-working pillar.example is unnecessary because it requires me to keep a fork, raise PR and obtain an appreciation of Tomcat formula business logic at a time when I'm more concerned with deployment UC - getting the software and formula out there so end-user community can revert with feedback which then feeds into *good defaults* for custom pillars. 

I agree with your other technical comments. Will implement.